### ISSUE TITLE
 Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work

### ISSUE THREAD

**Issue by l00kanio** [oxtyped/gpodder2go#14]
I tried to run binary on my arm64 (armbian) and then x86-64 (manjaro) machines. Unfortunately, every time I am getting error like:
```
./gpodder2go init
2023/09/27 23:41:09 Binary was compiled with 'CGO_ENABLED=0', go-sqlite3 requires cgo to work. This is a stub
```
Is there anything wrong with my systems or with binaries?  Any idea? I am not into Go in any way :-(

**Comment by oxtyped** [comment]
I've just released `v0.1.2` which contains the fix by @sbinet in #6 that removes the need for CGO.

Do give it a try and let us know if you're still running into any issues https://github.com/oxtyped/gpodder2go/releases

**Comment by l00kanio** [comment]
Yes, it works! (Tested on arm64, so far). Thank you for developing such useful piece of software
